### ISSUE TITLE
.hadolint.yml is not detected but .hadolint.yaml is

### ISSUE THREAD

**Issue by carlsmedstad** [hadolint/hadolint#629]
- [x] This is a bug report
- [ ] This is a feature request
- [ ] I searched existing issues before opening this one

### Expected behavior
`.hadolint.yml` in project root is detected or the README does not state that it should be.

### Actual behavior
`.hadolint.yml` is not detected but `.hadolint.yaml` is.

### Steps to reproduce the behavior
Config:

```yaml
# file: .hadolint.yml
ignored:
  - DL3059
```

```sh
$ hadolint Dockerfile
Dockerfile:4 DL3059 info: Multiple consecutive `RUN` instructions. Consider consolidation.
```

**Dockerfile**

```Dockerfile
FROM centos:7

RUN echo Hello
RUN echo Hello
```

**Additional environment details (OS, stack version, etc.)**
Tested with image `hadolint/hadolint:latest-debian` and natively on Arch Linux.
```sh
$ hadolint --version
Haskell Dockerfile Linter 2.3.0-no-git
```

**Comment by lorenzo** [comment]
I think this is fixed since 2.4.0. Please reopen if that is not the case for you

**Comment by jokay** [comment]
@lorenzo I think this problem is back in **2.4.1-0-g4b95e40**.
